data/builtin_mount_options.conf

```diff
@@ -8,8 +8,14 @@ vfat_allow=uid=$UID,gid=$GID,flush,utf8,shortname,umask,dmask,fmask,codepage,ioc
 exfat_defaults=uid=$UID,gid=$GID,iocharset=utf8,errors=remount-ro
 exfat_allow=uid=$UID,gid=$GID,dmask,errors,fmask,iocharset,namecase,umask
 
+# the legacy ntfs kernel driver and the ntfs-3g fuse driver
 ntfs_defaults=uid=$UID,gid=$GID,windows_names
```

Comment:
I think adding the `ntfs:` prefix here would make it easier to understand: `ntfs:ntfs_defaults` and `ntfs:ntfs_allow`.